### PR TITLE
docs(popover): explain how to pop on the correct side

### PR DESCRIPTION
**Pull request by Kiarokh**
fix: https://github.com/Lundalogik/crm-feature/issues/1849
## Review:
- [ ] Commits are [atomic](https://seesparkbox.com/foundry/atomic_commits_with_git)
- [ ] Commits have the correct *type* for the changes made
- [ ] Commits with *breaking changes* are marked as such

### Browsers tested:
(Check any that applies, it's ok to leave boxes unchecked if testing something didn't seem relevant.)

Windows:
- [ ] Chrome
- [ ] Edge
- [ ] Firefox

Linux:
- [ ] Chrome
- [ ] Firefox

macOS:
- [ ] Chrome
- [ ] Firefox
- [ ] Safari

Mobile:
- [ ] Chrome on Android
- [ ] iOS

### CODE REVIEW

File: src/components/popover/popover.tsx [hunk 1]
```diff
@@ -39,6 +39,16 @@ import { portalContains } from '../portal/contains';
  * When a popover is displayed, interactions with other controls are blocked,
  * until user dismisses the popover.
  *
+ * :::warning
+ * In order for the popover to pop on the correct side of the element that
+ * triggers it, you need to wrap both of them in a container and make sure that
+ * the container:
+ * 1. is visually as big as the element that triggers the popover
```
Comment (adrianschmidt) [inline src/components/popover/popover.tsx:46]: Can't we provide the container ourselves, and put the trigger in a slot. I'm pretty sure that's what we do in limel-menu, and that works just fine…

Then we wouldn't need this big warning.

Comment (maltja) [reply src/components/popover/popover.tsx:46]: We also thought about that. So maybe we should try to do that instead 🙂

Comment (Kiarokh) [reply src/components/popover/popover.tsx:46]: I don't know how or where we should do this :/ can you suggest a solution in code?

Comment (maltja) [reply src/components/popover/popover.tsx:46]: I can try to do it 🙂

Comment (adrianschmidt) [reply src/components/popover/popover.tsx:46]: Give me a holler tomorrow if you don't manage, but if you look at limel-menu, I think it'll be pretty straightforward 👍

**Comment by adrianschmidt** [comment]
I'll be adding a different solution to this problem.

**Pull request closed without merging**